src/Language/Spectacle/Exception/RuntimeException.hs

```diff
@@ -13,8 +13,10 @@ import Type.Reflection (Typeable)
 
 data RuntimeException where
   VariableException :: VariableException -> RuntimeException
-  deriving (Exception, Show, Typeable)
+  deriving (Show, Typeable)
```

Comment:
You need to say `deriving stock` here, otherwise without an explicit strategy, it's just the default/`NoDerivingStrategies` behavior.

Reply:
Isn't the default deriving strategy for stock typeclasses just stock anyway?

Reply:
Yes! So technically omitting `stock` has the same behavior in this case.

But that's not being enforced in the code; you're just knowledgeable enough to know which typeclasses would have used the `stock` strategy implicitly. The point of specifying them is to make things explicit, so GHC keeps you honest, and developers can more easily read the code.

Reply:
Ah, gotcha! Since I culled this branch already I'm going to push this change into #9.

Reply:
Thanks for taking a look over these btw!